e2e/cypress/support/index.js

```diff
@@ -13,6 +13,7 @@
 // https://on.cypress.io/configuration
 // ***********************************************************
 
+require('cypress-plugin-retries');
```

Comment:
this plugin just retries a single statement, not a complete "spec", right? Maybe https://github.com/bahmutov/cypress-retry will yield better results...

Reply:
okay, using cypress-retry will have it's own start-up file again and the action won't be usable